### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by avijayakumarmib**
Updated read me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Public Repository
 =================
 
-Testing public repositories.
+Testing public repositories in GitHub.
```
Comment (krloxz) [inline README.md:4]: Comment abc!

File: README.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 Public Repository
 =================
```
Comment (krloxz) [inline README.md:2]: Comment xyz

File: README.md (hunk 1 above)
Comment (avijayakumarmib) [reply README.md:4]: reply for comment abc

File: README.md (hunk 2 above)
Comment (avijayakumarmib) [reply README.md:2]: reply comment for xyz

**Review by krloxz — APPROVED** [review]
Approved

**Pull request merged**